src/Components/Web/src/Virtualization/PlaceholderContext.cs

```diff
@@ -13,13 +13,20 @@ namespace Microsoft.AspNetCore.Components.Web.Virtualization
         /// </summary>
         public int Index { get; }
 
+        /// <summary>
+        /// The size of the placeholder in pixels.
+        /// </summary>
+        public float Size { get; }
```

Comment:
Should this be `Height`?

Reply:
I named it `Size` since we could support horizontal lists in the future, and it's also consistent with `ItemSize`.

Reply:
Could you expand the description in that case and say what this would mean? 

```
<para>
For virtualized components with vertical scrolling, this would be the height of the placeholder in pixels. For virtualized components with horizontal scrolling, this would be the width of the placeholder in pixels.
</para>
```